pxf/pxf-hive/src/main/java/org/apache/hawq/pxf/plugins/hive/utilities/EnumHiveToHawqType.java

```diff
@@ -110,4 +122,68 @@ public static EnumHiveToHawqType getHiveToHawqType(String hiveType) {
                 + hiveType + " to HAWQ's type");
     }
 
+
+    /**
+     * 
+     * @param dataType Hawq data type
+     * @return compatible Hive type to given Hawq type, if there are more than one compatible types, it returns one with bigger size
+     * @throws UnsupportedTypeException if there is no corresponding Hive type for given Hawq type
+     */
+    public static EnumHiveToHawqType getCompatibleHawqToHiveType(DataType dataType) {
+
+        SortedSet<EnumHiveToHawqType> types = new TreeSet<EnumHiveToHawqType>(
+                new Comparator<EnumHiveToHawqType>() {
+                    public int compare(EnumHiveToHawqType a,
+                            EnumHiveToHawqType b) {
+                        return Byte.compare(a.getSize(), b.getSize());
+                    }
+                });
```

Comment:
Can this `types` variable be instantiated outside the method? It doesn't changed based on input right?